ShinyExpenseTracker/src/ca/ualberta/cs/shinyexpensetracker/activities/AddDestinationActivity.java

```diff
@@ -37,6 +38,8 @@
  */
 public class AddDestinationActivity extends Activity {
 	public static final String DESTINATION_INDEX = "destinationIndex";
+	public static final int SET_GEOLOCATION = 1;
+	private static final Coordinate NORTH_KOREA_CONCENTRATION_CAMP_COORDINATES = new Coordinate(39.03808, 125.7296);
```

Comment:
This default coordinate set should defined only once, in some other file. The Coordinate class itself, perhaps?